assets/base.css

```diff
@@ -1108,7 +1129,6 @@ summary::-webkit-details-marker {
   }
 
   .grid--peek .grid__item {
-    width: calc(50% - var(--grid-mobile-horizontal-spacing) - 3rem);
     min-width: 35%;
   }
```

Comment:
Featured blog and main product dont have the settings we are applying here. It looks like we don't need the width, but the min width is actually being used: https://screenshot.click/23-15-7e58n-ac2m8.png

Reply:
However I wonder if we should keep it for consistency since this is a class with no modifier and if we were to add it to a new slider it would make the most sense to already have it there so a dev would know what to expect in a slider when applying the class.

Reply:
We also have this on it though: 
```css
.product__media-list .product__media-item {
    width: calc(100% - 3rem);
}
```
So I'm not sure when that min-width comes in handy 🤔

Reply:
This is only for the product media list. I mean if we create any new sections that use grid peek and grid item, itd be best if they come from here instead of from the modifiers. Does that make sense? 🤔 

Perhaps we can create a follow up to clean up some of the existing code. Just not sure if it makes sense to remove it in this PR since it's already on main. What do you think?

Reply:
As far as the removing the `width` goes. I could see removing it or not. It makes sense why it's there, serving as a default. Theoretically, we shouldn't have to apply a 2-col class if we wanted the slider to be 2 cols, but we always do. For one line of code though, I'd rather play it safe and leave it there.

Reply:
Ah didn't realize the min-width was fixing a bug. Then let's not touch anything here 👍

Reply:
I would rather keep it too to ensure we dont break anything. We can create a follow up to clean up the code if needed 👍